src/test/java/org/corfudb/infrastructure/ManagementServerTest.java

```diff
@@ -0,0 +1,35 @@
+package org.corfudb.infrastructure;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Checks the various services and messages handled by the
+ * Management Server.
+ * <p>
+ * Created by zlokhandwala on 11/2/16.
+ */
+public class ManagementServerTest extends AbstractServerTest {
+
+    @Override
+    public ManagementServer getDefaultServer() {
+        ServerContext serverContext = new ServerContextBuilder()
+                .setSingle(false)
+                .setServerRouter(getRouter())
+                .build();
+        return new ManagementServer(serverContext, new LayoutServer(serverContext));
+    }
+
+    /**
+     * Testing the status of the failure detector and shutdown functionality.
+     */
+    @Test
+    public void checkFailureDetectorStatus() {
```

Comment:
Yes it is meant to check the shutdown function only.
The pollingPolicy has its own test. I felt a more detailed test can be written once other modules are in place.
I did not want to expose unrelated data from the policy just for the test.